Description: Disable bench tests so that autopkgtest won't fail
 bench tests can't be run with a stable rust
Forwarded: not-needed
Last-Update: 2023-02-23
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -29,6 +29,7 @@
 ]
 license = "MIT OR Apache-2.0"
 repository = "http://github.com/ejmahler/strength_reduce"
+autobenches = false
 
 [dev-dependencies.num-bigint]
 version = "0.4"
